src/pages/tags/index.astro

```diff
@@ -2,22 +2,59 @@
 import { getCollection } from "astro:content";
 
 import Layout from "@/layouts/base.astro";
+import { getAuthorOrDefault } from "@/modules/author";
 
 // Fetch all documents from the 'tags' collection
 const tags = await getCollection("tags");
 ---
 
-<Layout class="p-6">
-    <div class="space-y-4">
+<content>
+    <title>RBX docs</title>
+    <meta content="Rbx Docs" property="og:title" />
+    <meta content="Tags helpful for roblox development" property="og:description" />
+    <meta content={`/RbxDocs/tags`} property="og:url" />
+    <meta content="#4c51bf" data-react-helmet="true" name="theme-color" />
+</content>
+
+<Layout class="px-4 py-8 md:px-16 lg:px-72">
+    <div class="elements">
         {
-            tags.map(doc => (
-                <a
-                    href={`/RbxDocs/tags/${doc.slug}`}
-                    class="inline-block rounded bg-blue-500 px-4 py-2 text-white hover:bg-blue-600"
+            tags.map(async doc => (
```

Comment:
This shouldn't work, is there something I'm missing here? Using a promise as a mapping function causes an array of promises to be returned rather than the intended effect, and requires [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) to be used alongside.

Reply:
Well it works so it seems like astro can detect promises or something.